ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/ApiServices.scala

```diff
@@ -149,6 +139,8 @@ object ApiServices {
 
       val apiReflectionService = ProtoReflectionService.newInstance()
 
+      val authorizer = new Authorizer(TimeProvider.clock(timeProvider))
```

Comment:
This looks like the clock for checking token expiration is set to be identical to the clock used for DAML transactions. Is this a good idea in the case of a ledger in static time mode? I would have imagined that token expiration always refers to UTC time.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3286/commits/0799c10dc275376fa91a59be992692f4ccaa6944